website/docs/r/cos_bucket.html.markdown

```diff
@@ -221,26 +284,48 @@ Review the argument references that you can specify for your resource.
 - `archive_rule` - (Required, List) Nested archive_rule block has following structure.
   
   Nested scheme for `archive_rule`:
-  - `days` - (Required, String) Specifies the number of days when the specific rule action takes effect.
+  - `days` - (Required, Integer) Specifies the number of days when the specific rule action takes effect.
   - `enable` - (Required, Bool) Specifies archive rule status either `enable` or `disable` for a bucket.
   - `rule_id` -  (Optional, Computed, String) The unique ID for the rule. Archive rules allow you to set a specific time frame after the objects transition to the archive.
   - `type` - (Required, String) Specifies the storage class or archive type to which you want the object to transition. Allowed values are `Glacier` or `Accelerated`. 
   
-    **Note:** Archive is available in certain regions only. For more information, see [Integrated Services](https://cloud.ibm.com/docs/cloud-object-storage/basics?topic=cloud-object-storage-service-availability).
-- `expire_rule` - (Required, List) Nested expire_rule block has following structure.
+    **Note:** 
+    - Archive is available in certain regions only. For more information, see [Integrated Services](https://cloud.ibm.com/docs/cloud-object-storage/basics?topic=cloud-object-storage-service-availability).
+    - Restoring object once archive is not supported yet.
+- `expire_rule` - (Required, List) An expiration rule deletes objects after a defined period (from the object creation date). see [lifecycle actions](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-versioning). Nested expire_rule block has following structure.
   
   Nested scheme for `expire_rule`:
   - `rule_id` -  (Optional, Computed, String) Unique ID for the rule. Expire rules allow you to set a specific time frame after which objects are deleted.
   - `enable` - (Required, Bool) Specifies expire rule status either `enable` or `disable` for a bucket.
-  - `days` - (Required, String) Specifies the number of days when the specific rule action takes effect.
```

Comment:
Arrange the nested structure inner attributes in alphabetical order for all rules